components/klaudsolcms/modals/modal_body/CollectionTypeBody.js

```diff
@@ -58,31 +60,35 @@ export default function CollectionTypeBody({ formRef }) {
       <Formik {...formikParams}>
         <Form>
           <div>
-            <div className="d-flex justify-content-between align-items-center">
-              <h6 className="mx-3"> Configurations </h6>
-              <div>
-                <button className="btn_modal_settings"> Basic settings </button>
-              </div>
-            </div>
-            <div className="block_bar" />
-            <div className="row">
-              <div className="col">
-                <p className="mt-2"> Display Name </p>
-                <Field type="text" className="input_text" name="name" />
+              <label className="general-checkbox">
+                <Field
+                  type="checkbox"
+                  name="is_single_type"
+                  className="general-checkbox-box"
+                />
+                <span className="general-checkbox-text">
+                  Single Type
+                </span>
+              </label>
```

Comment:
Can we make this into a dropdown? I still have some variants in mind (I'm running out of synonyms for "types" :P).

The menu should be:

* Collection (default)
* Singleton
* Table-based (do not include yet, it's a future feature)